0.preprocess-sites/2.process-cells.py

```diff
@@ -174,11 +168,14 @@
     ), "Stop! You're counting cells more than once"
 
     # Create a summary of counts of each cell quality class
-    cell_count_df = pd.DataFrame(
-        metadata_df.Cell_Class.value_counts()
-        .rename(columns={"Cell_Class": "cell_count"})
-        .assign(site=site)
-    )
+    cell_count_df = pd.DataFrame(metadata_df.Cell_Class.value_counts())
+    cell_count_df.rename(columns={"Cell_Class": "cell_count"}).assign(site=site)
+
+    output_folder = pathlib.Path(output_basedir, batch, "paint", site)
+    os.makedirs(output_folder, exist_ok=True)
+
+    meta_output_file = pathlib.Path(output_folder, f"metadata_{site}.tsv.gz")
+    count_output_file = pathlib.Path(output_folder, f"cell_counts_{site}.tsv")
```

Comment:
I'm not sure the Pythonic reason why  "Create a summary of counts of each cell quality class" rows 177-181 needed to be split into two steps to work, but it's what I had to do to make the script run this time through...

I moved the location of the output folder creation so that we didn't create empty folders for sites that weren't able to be processed. This helps downstream recipe steps that create lists of sites to process by looking at the folder names.